examples/credential_issuer_metadata_jwt_vc_json.json

```diff
@@ -4,6 +4,14 @@
     "credential_endpoint": "https://credential-issuer.example.com",
     "batch_credential_endpoint": "https://credential-issuer.example.com/batch_credential",
     "deferred_credential_endpoint": "https://credential-issuer.example.com/deferred_credential",
+    "credential_response_encryption": {
+        "alg_values_supported" : [
+            "ECDH-ES"
+        ],
+        "enc_values_supported" : [
+            "A128GCM"
+        ]
```

Comment:
```suggestion
        "enc_values_supported" : [
            "A128GCM"
        ],
        "jwk_values_supported": [
            { "kty": "EC", "crv": "P-256" },
            { "kty": "EC", "crv": "X25519" },
            { "kty": "RSA" }            
        ],
```

Reply:
Only if people believe this is useful. See description of `jwk_values_supported` below.

Reply:
kty for X25519 is OKP FWIW https://www.rfc-editor.org/rfc/rfc8037.html#section-2

Reply:
@selfissued is this also something that we could solve with fully-specified algorithms? To me the solution proposed by Oliver looks rather ugly. Unfortunately Mikes proposal only targets signatures and not yet encyprtion

Reply:
@awoie to file a separate issue.

Reply:
Done, see #168